Source/Request/Request.JSONP.js

```diff
@@ -119,19 +119,20 @@ Request.JSONP = new Class({
 		return !!this.running;
 	},
 
-	clear: function(){
+	clear: function(index){
```

Comment:
it's not breaking because worst case scenario, it will just try to delete an undefined property. 

I have a better fix which does not change the API - storing the index within the script tag, then auto deleting

http://jsfiddle.net/dimitar/frxnqj5q/1/